app/static/src/app/store/fitData/state.ts

```diff
@@ -0,0 +1,8 @@
+import { Dict } from "../../types/utilTypes";
```

Comment:
Why have this as a separate file from `fitData.ts`?

Reply:
You get a circular dependency otherwise, since actions and mutations import the state interface, and `fitData` imports actions and mutations.